x-pack/plugins/cases/public/components/case_view/case_view_page.tsx

```diff
@@ -39,12 +47,7 @@ export const CaseViewPage = React.memo<CaseViewPageProps>(
 
     useCasesTitleBreadcrumbs(caseData.title);
 
-    const activeTabId = useMemo(() => {
-      if (urlParams.tabId && Object.values(CASE_VIEW_PAGE_TABS).includes(urlParams.tabId)) {
-        return urlParams.tabId;
-      }
-      return CASE_VIEW_PAGE_TABS.ACTIVITY;
-    }, [urlParams.tabId]);
+    const activeTabId = getActiveTabId(urlParams?.tabId);
```

Comment:
When I removed some tests the other tests started failing. There was an issue with mocking and memorization. I removed the `useMemo` as it is redundant.